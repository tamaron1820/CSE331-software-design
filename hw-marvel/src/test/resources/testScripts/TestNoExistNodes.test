# Test if the nodes are not existed in graph

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/soccerPlayer.csv

LoadGraph soccer soccerPlayer.csv

FindPath soccer Messi Ronaldo
FindPath soccer Kevin-De-bruyne Neymar