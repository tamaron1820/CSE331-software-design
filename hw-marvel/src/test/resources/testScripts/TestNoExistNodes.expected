# Test if the nodes are not existed in graph

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/soccerPlayer.csv

loaded graph soccer

unknown: Ronaldo
unknown: Kevin-De-bruyne